### DISCUSSION TITLE
Exterior/Math/P%26S/9/

### DISCUSSION THREAD

**Discussion by giscus** [SAVIA7582/Exterior#12] [General]
# Exterior/Math/P%26S/9/

https://savia7582.github.io/Exterior/Math/P%26S/9/

**Comment by LH-0124** [comment]
F分布的上\alpha分位数性质有误

**Reply by NoughtQ** [reply]
？F分布的这个应该是对的呀，我发现的问题是上 $\alpha$ 分位数的定义写错了，应该是“当 $P(Y > y_\alpha) = \alpha$ 时，称 $y_\alpha$ 为该分布的上 $\alpha$ 分位数”

**Reply by LH-0124** [reply]
对的是因为savia改过了哈哈哈哈

**Comment by LH-0124** [comment]
区间估计和假设检验两个正态总体的枢轴量和检验统计量均有误，分子分母的系数在F分布中应该被除掉了

**Reply by LH-0124** [reply]
是针对方差之比的那一条

**Reply by LH-0124** [reply]
savia已更正

**Comment by WhereIsMyCake** [comment]
第2页区间估计的基本原理那里，枢轴量是不是应该满足t(n)分布捏

**Reply by Kyrie-1124** [reply]
写错了吧，应该是t(n-1)分布

**Comment by WhereIsMyCake** [comment]
还有第4页（1）的解，答案没有问题，但是过程中应该写 $\chi_{0.95}^2(15)$ 吧？

**Reply by Kyrie-1124** [reply]
写错了这块，应该把所有n替换为n-1并且16替换为15，自由度肯定是15

**Comment by SAVIA7582** [comment]
紧急纠错：在导出置信区间和拒绝域时，如果枢轴量满足的分布是x(n-1)，则应该将n-1代入而非n。原讲义中没有考虑这一点，后面会用更恰当的语言来描述

**Comment by Hzw-ZuttoZutto** [comment]
拟合优度检验部分中卡法分布的自由度是否最好写成k-r-1, r是分布未知的参数数量，不然与对应例题对不上

**Reply by SAVIA7582** [reply]
好的，我回头看看，不过好像听说这次明确不考拟合优度检验
